src/js/actions/search/commands.js

```diff
@@ -50,18 +50,20 @@ define(function (require, exports) {
             menuTree;
 
         parts.forEach(function (part) {
-            nlsPath = nlsPath + "." + part;
-            menuTree = nls.localize(nlsPath);
+            if (!_.isNumber(parseInt(part, 10))) {
```

Comment:
There is no `else` statement here, do we not need to append the number itself to the resultPath at the end?